src/components/Calculator.css

```diff
@@ -0,0 +1,24 @@
+.button-wrapper {
+  height: 4em;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-weight: lighter;
+  font-size: 1.4em;
+  background-color: #e0e1e6;
+  color: rgb(49, 49, 49);
+  flex: 1;
+  outline: 1px solid #888;
+}
+
+.operator {
+  background-color: #fe9241;
+  display: flex;
+  flex: 1;
+  color: #fff;
+}
+
+.input-calc {
+  justify-items: right;
+  width: 100%;
+}
```

Comment:
- [ ] Try to follow the given design

Design
![design](https://user-images.githubusercontent.com/44745855/128942463-b88e9eda-143e-4574-ac94-ce2a6b4ee6dd.png)

yours
![yr](https://user-images.githubusercontent.com/44745855/128942485-e49285a3-50c3-4b15-93c7-daa231d9f014.png)